atomic_reactor/plugins/exit_store_metadata.py

```diff
@@ -61,46 +61,40 @@ def get_digests(self):
         """
 
         digests = {}  # repository -> digest
-        for registry in self.workflow.push_conf.docker_registries:
+        for registry, registry_conf in self.workflow.conf.registries.items():
```

Comment:
Is it the case that 'registry' is unused here?

I.e. `for _, registry_conf in self.workflow.conf.registries.items():` would also work?

Reply:
updated